c_binding/include/libstoragemgmt/libstoragemgmt_types.h

```diff
@@ -340,6 +372,56 @@ typedef enum {
 #define LSM_SYSTEM_CACHE_PCT_NO_SUPPORT             -2
 #define LSM_SYSTEM_CACHE_PCT_UNKNOWN                -1
 
+#define LSM_VOLUME_WRITE_CACHE_POLICY_UNKNOWN       0
```

Comment:
I intend to pick `1 (1 << 0)` for UNKNOWN and `2 (1 << 2)` for NO_SUPPORT in order to support bitmap used in status.